profiles/api/webhook.py

```diff
@@ -0,0 +1,29 @@
+from flask import Blueprint
+from werkzeug.exceptions import NotFound
+from werkzeug.wrappers import Response
+
+from profiles.commands import update_profile_from_orcid_record
+from profiles.exceptions import ProfileNotFound
+from profiles.orcid import OrcidClient
+from profiles.repositories import OrcidTokens, Profiles
+
+
+def create_blueprint(profiles: Profiles, orcid_client: OrcidClient,
+                     orcid_tokens: OrcidTokens) -> Blueprint:
+    blueprint = Blueprint('webhook', __name__)
+
+    @blueprint.route('/orcid-webhook/<orcid>', methods=['POST'])
+    def _update(orcid: str) -> Response:
+        try:
+            profile = profiles.get_by_orcid(orcid)
+        except ProfileNotFound as exception:
+            raise NotFound(str(exception)) from exception
+
+        access_token = orcid_tokens.get(profile.orcid)
+        orcid_record = orcid_client.get_record(orcid, access_token.access_token)
```

Comment:
Need to cater for the access token being rejected.

Reply:
What status code would you want returned in the event of a token rejection?

Reply:
It's what ORCID would reject us with. Guessing it's a 403?